### ISSUE TITLE
qsv frequency --ignore-case

### ISSUE THREAD

**Issue by tmtmtmtm** [jqnatividad/qsv#1380]
It would sometimes be very useful to be able to ignore case when constructing frequency tables, but as far as I can see, `qsv frequency` doesn't currently take such an option, and I'm struggling to come up with a way to emulate this either, other than brute-forcing everything to either lower or upper-case first which is far from ideal.

**Comment by jqnatividad** [comment]
I agree that this will be a good addition to `frequency`.  Adding to the backlog.
